File: tutorials/30-operators.py
# Day 2: Operators
# Start using arithmetic operators.
#
# https://www.hackerrank.com/challenges/30-operators/problem
#

#!/bin/python3

import sys

if __name__ == "__main__":
    meal_cost = float(input().strip())
    tip_percent = int(input().strip())
    tax_percent = int(input().strip())

    cost = meal_cost * (1 + tip_percent / 100 + tax_percent / 100)

    print("The total meal cost is {:.0f} dollars.".format(cost))

File: tutorials/30-days-of-code/30-operators.cpp
// Day 2: Operators
// Start using arithmetic operators.
//
// https://www.hackerrank.com/challenges/30-operators/problem
//

#include <bits/stdc++.h>

using namespace std;

int main() {
    double meal_cost;
    cin >> meal_cost;
    int tip_percent;
    cin >> tip_percent;
    int tax_percent;
    cin >> tax_percent;

    cout << "The total meal cost is "
         << round(meal_cost * (1 + tip_percent / 100. + tax_percent / 100.))
         << " dollars." << endl;

    return 0;
}
